platform/src/main/java/org/stellar/anchor/platform/observer/stellar/StellarPaymentObserver.java

```diff
@@ -360,13 +361,14 @@ void handleEvent(OperationResponse operationResponse) {
 
     ObservedPayment observedPayment = null;
     try {
-      if (operationResponse instanceof PaymentOperationResponse) {
-        PaymentOperationResponse payment = (PaymentOperationResponse) operationResponse;
+      if (operationResponse instanceof PaymentOperationResponse payment) {
         observedPayment = ObservedPayment.fromPaymentOperationResponse(payment);
-      } else if (operationResponse instanceof PathPaymentBaseOperationResponse) {
-        PathPaymentBaseOperationResponse pathPayment =
-            (PathPaymentBaseOperationResponse) operationResponse;
+      } else if (operationResponse instanceof PathPaymentBaseOperationResponse pathPayment) {
         observedPayment = ObservedPayment.fromPathPaymentOperationResponse(pathPayment);
+      } else if (operationResponse
+          instanceof InvokeHostFunctionOperationResponse invokeHostFunction) {
+        observedPayment =
```

Comment:
It would be nice to add some background information of this type of response.

Reply:
Will document when we implement it for real